lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -223,17 +222,8 @@ newTransactionLayer networkId = TransactionLayer
     }
   where
     _initDelegationSelection
-        :: FeePolicy
-            -- Current fee policy
-        -> DelegationAction
-            -- What sort of action is going on
-        -> CoinSelection
-        -- ^ An initial selection where 'deposit' and/or 'reclaim' have been set
-        -- accordingly.
-    _initDelegationSelection (LinearFee _ _ (Quantity c)) = \case
-        Quit{} -> mempty { reclaim = round c }
-        Join{} -> mempty
-        RegisterKeyAndJoin{} -> mempty { deposit = round c }
+        :: CoinSelection
+    _initDelegationSelection = mempty
```

Comment:
I am not sure to understand this change? what happened to the reclaim and the deposit needed for the coin selection to run correctly?

Reply:
I'm also wondering why I did that. But what's even more interesting is why the tests pass

Reply:
Ah, now I remember... we removed the 3rd parameter for LinearFee https://github.com/input-output-hk/cardano-wallet/pull/2327/files#diff-45c6b9a78752ca17442ffe1ded898e5fd2f6931f95c9ae4ad21d30afcf0e2e1fL810

Reply:
Yes okay, the third parameter was removed from the linear fee, but it's because it now comes from the protocol parameters directly as the stake key deposit. The logic that applied here still applies however!